### PR TITLE
Bigtable quick start

### PR DESCRIPTION
**Pull request by AVaksman**
Adding a Cloud Bigtable QuickStart project to demonstrate basic operations with Cloud Bigtable.

### CODE REVIEW

File: bigtable/api/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# .NET Cloud Bigtable Samples
+
+A collection of samples that demonstrate how to call the
+[Google Cloud Bigtable API](https://cloud.google.com/bigtable/docs/) from C#.
+
+    This sample requires [.NET Core 2.0](
+        https://www.microsoft.com/net/core) or later.  That means using
+    [Visual Studio 2017](
+        https://www.visualstudio.com/), or the command line.  Visual Studio 2015 users
```
Comment (AVaksman) [inline bigtable/api/README.md]: Take out beginning of the next sentence. "Visual Studio 2015 users"

File: bigtable/api/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+5.  Provisioning an instance 
+    Follow the instructions in the [user
+    documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+    create a Cloud Bigtable instance if necessary.
+
+7.  Edit `QuickStart\Quickstart.cs`, and replace YOUR-PROJECT-ID with id
+    of the project you created in step 1. 
+    Also replace YOUR-INDEX-ID with id of your instance you created in step 5.
+
```
Comment (AVaksman) [inline bigtable/api/README.md]: "YOUR-INSTANCE-ID"

**Comment by jabubake** [comment]
Please rename this current sample to "Hello world" since there are more operations performed than a "quick start".

https://github.com/jabubake/nodejs-bigtable/blob/master/samples/README.md#quickstart
Please see expected quickstart for Bigtable here :
 https://github.com/jabubake/nodejs-bigtable/blob/master/samples/quickstart.js
(Read a row from an existing table)

**Comment by AVaksman** [comment]
Yes I saw the "quick start" for Node.js.
Shall I leave the Hello World example in this repository as it already present in [cloud-bigtable-examples](https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/tree/master/dotnet/hello_world) repository?
I will follow your example of quick start as well as examples for instance and table management

**Comment by jabubake** [comment]
It seems worth keeping all the samples bundled in one location. 
Lets keep the samples in this repo (aka move hello world here).

**Comment by AVaksman** [comment]
Closing this PR as [#560](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/560) includes all samples for Bigtable

**Pull request closed without merging**